subsys/bluetooth/controller/hci_internal.c

```diff
@@ -28,7 +28,7 @@
 static struct
 {
 	bool occurred; /**< Set in only one execution context */
-	uint8_t raw_event[CONFIG_BT_RX_BUF_LEN];
+	uint8_t raw_event[CONFIG_BT_BUF_ACL_RX_SIZE + 4];
```

Comment:
This buffer is used to store command complete events, not ACL data. We therefore need to size the buffer accordingly.
```suggestion
	/* HCI_Read_Supported_Commands is the command which returns the largest command
	 * command complete event.
	 */
	uint8_t raw_event[struct(struct bt_hci_rp_read_supported_commands) + BT_HCI_EVT_HDR_SIZE];
```

Reply:
I would suggest to use `BT_BUF_EVT_RX_SIZE` instead.

If no the above suggestion has a mistake (missing rest of command complete) and should be updated to:
`sizeof(struct bt_hci_rp_read_supported_commands) + CMD_COMPLETE_MIN_SIZE`